### PR TITLE
Balances welder fuel capacity

### PR DESCRIPTION
**Pull request by endmauler**
## About The Pull Request
Halves the fuel capacity of all welders.
## Why It's Good For The Game
It balances the welder's fuel capacity.
## Changelog
:cl:
:tweak: Halved the fuel capacity of all welders
/:cl:

### CODE REVIEW

**Comment by 4dplanner** [comment]
lol

**Comment by nemvar** [comment]
Welder already empty pretty fast if you use them frequently. If you want to adjust something about them, look at the recharge rate of the experimental welder or make them not filled up upon printing. I think welder balance is in a good place right now. It's something you have to keep in the back of your mind if you plan on doing lots of construction, but is never oppressive.

While I'm quick to make an argument for nerfs, I don't quite see the point in this one.

**Comment by PKPenguin321** [comment]
Actual reasoning?

**Comment by 81Denton** [comment]
Why is it good for the game though

**Comment by Barhandar** [comment]
Perfectly balanced, as all things should be.

**Comment by Fikou** [comment]
ided

**Comment by zxaber** [comment]
I suppose I'll just head to cargo to print out a box of disposable welders when I'm a roboticist.

File: code/game/objects/items/tools/weldingtool.dm [hunk 1]
```diff
@@ -24,7 +24,7 @@
 	materials = list(MAT_METAL=70, MAT_GLASS=30)
 	var/welding = 0 	//Whether or not the welding tool is off(0), on(1) or currently welding(2)
 	var/status = TRUE 		//Whether the welder is secured or unsecured (able to attach rods to it to make a flamethrower)
-	var/max_fuel = 20 	//The max amount of fuel the welder can hold
+	var/max_fuel = 10 	//The max amount of fuel the welder can hold
```
Comment (Jalleo) [inline code/game/objects/items/tools/weldingtool.dm]: I suggest 15

File: code/game/objects/items/tools/weldingtool.dm [hunk 2]
```diff
@@ -296,7 +296,7 @@
 	name = "industrial welding tool"
 	desc = "A slightly larger welder with a larger tank."
 	icon_state = "indwelder"
-	max_fuel = 40
+	max_fuel = 20
```
Comment (Jalleo) [inline code/game/objects/items/tools/weldingtool.dm]: 30

File: code/game/objects/items/tools/weldingtool.dm [hunk 3]
```diff
@@ -317,7 +317,7 @@
 	name = "emergency welding tool"
 	desc = "A miniature welder used during emergencies."
 	icon_state = "miniwelder"
-	max_fuel = 10
+	max_fuel = 5
```
Comment (Jalleo) [inline code/game/objects/items/tools/weldingtool.dm]: This is too low. You cant use it.

File: code/game/objects/items/tools/weldingtool.dm [hunk 4]
```diff
@@ -344,15 +344,15 @@
 	desc = "An upgraded welder based of the industrial welder."
 	icon_state = "upindwelder"
 	item_state = "upindwelder"
-	max_fuel = 80
+	max_fuel = 40
```
Comment (Jalleo) [inline code/game/objects/items/tools/weldingtool.dm]: This looks good.

File: code/game/objects/items/tools/weldingtool.dm [hunk 5]
```diff
@@ -344,15 +344,15 @@
...
 	materials = list(MAT_METAL=70, MAT_GLASS=120)
 
 /obj/item/weldingtool/experimental
 	name = "experimental welding tool"
 	desc = "An experimental welder capable of self-fuel generation and less harmful to the eyes."
 	icon_state = "exwelder"
 	item_state = "exwelder"
-	max_fuel = 40
+	max_fuel = 20
```
Comment (Jalleo) [inline code/game/objects/items/tools/weldingtool.dm]: Remove it entirely here. Make it the same as a normal welding tool in amount.

**Review by Jalleo — CHANGES_REQUESTED** [review]
My opinion. Its good to a degree.

**Comment by Jalleo** [comment]
This is a good partial counterbalance to selling all the welding fuel tanks. Other aspect is the autolathe fuel but that is a totally different issue.

**Comment by Michael-Ashfield** [comment]
This isn't needed. Welders aren't overpowered in their current form. If you wanna balance it just stop them being printed with fuel as this can sometimes be quite strong when fighting a blob.

**Comment by 81Denton** [comment]
If OP doesn't explain the reasoning behind this PR, I'm giving it the axe around the 24 hour mark

**Comment by PKPenguin321** [comment]
Thanos meme pr

**Pull request closed without merging**